internal/controller/isbservicerollout_controller.go

```diff
@@ -280,7 +281,7 @@ func (r *ISBServiceRolloutReconciler) processExistingISBService(ctx context.Cont
 	// if I need to update or am in the middle of an update of the ISBService, then I need to make sure all the Pipelines are pausing
 	isbServiceNeedsUpdating, isbServiceIsUpdating, err := r.isISBServiceUpdating(ctx, isbServiceRollout, existingISBServiceDef)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("error determining if ISBService is updating: %v", err)
```

Comment:
thanks for expanding on the errors with more info - I know I didn't do that in a lot of places